runWebpack.sh

```diff
@@ -136,6 +136,9 @@ fi
 # on the current APP environment - ugh
 #
 bash custom/customize.sh
+
+# see https://nodejs.org/api/cli.html#cli_max_old_space_size_size_in_megabytes
+export NODE_OPTIONS='--max-old-space-size=3584'
```

Comment:
Can we set it closer to 8g? Since the pod has a 8g limit? Or do we want to keep it at 4?

Reply:
i think it is still 4G in our current deployment https://github.com/uc-cdis/cloud-automation/blob/master/kube/services/portal/portal-deploy.yaml#L79